src/assets/scss/style.scss

```diff
@@ -202,8 +204,100 @@ textarea {
   font-weight: $font-weight-base;
 }
 
+// overrided btn-outline and primary as design spect for secondary buttons
+.btn-primary {
+  color: #ffffff;
```

Comment:
Do you know if there is any way in bootstrap to customize this so that we don't have to be specific to each button? 
I guess design wants something very custom

Reply:
I tried to use the same mixins or overriding variables but yes the requirements are very specific, the other option is tu use or own button prefix like liq-primary-btn, etc

Reply:
![Screen Shot 2020-12-11 at 10 33 34 AM](https://user-images.githubusercontent.com/1889891/101922671-62980900-3b9c-11eb-8ebc-0afe3af3562b.png)